docs/powerpoint/bind-shapes-in-presentation.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Bind to shapes in a PowerPoint presentation
+description: Learn how to bind shapes and access them from your add-in to keep them up-to-date.
+ms.topic: how-to
+ms.date: 03/31/2025
+ms.localizationpriority: medium
+---
+
+# Bind to shapes in a PowerPoint presentation
+
+Your PowerPoint add-in can bind to shapes to consistently access them through an identifier. Your add-in establishes a binding by calling `BindingCollection.add` and assigning a unique identifier. The identifier can be used at any time to reference the shape and access its properties. Creating bindings provides the following value to your add-in.
+
+- Establishes a relationship between the add-in and the shape in the document. Bindings are persisted in the document and can be accessed at a later time.
```

Comment:
```suggestion
- Establishes a relationship between the add-in and the shape in the document. Bindings are persistent in the document and can be accessed at a later time.
```
Persistent=adjective
Persisted=past tense verb

Reply:
I'm just taking localization into consideration here. It still seems to me that persistent is the better word choice.